<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplyWidget</class>
 <widget class="QWidget" name="ReplyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="2" column="0">
      <widget class="QCheckBox" name="autoFocusReplyCheck">
       <property name="toolTip">
        <string>鼠标移入通知时，自动聚集焦点至回复消息的输入框。
可能无法自动设置为输入焦点，
但可作为“鼠标移入时自动显示回复框”的效果来使用</string>
       </property>
       <property name="text">
        <string>自动聚焦回复框</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="autoShowReplyCheck">
       <property name="text">
        <string>自动展开回复框</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>快速打开最新消息的回复框
注意：会忽略按下快捷键前一瞬间收到的消息，回复x毫秒之前的最新消息</string>
       </property>
       <property name="text">
        <string>回复快捷键</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="AIReplyCheck">
       <property name="text">
        <string>AI回复参考</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="closeAfterReplyCheck">
       <property name="toolTip">
        <string>回复消息后快速隐藏，等待时间等同于失去焦点后保留显示的时长。
如果是快捷键打开回复，则自动关闭回复框，返回原先程序</string>
       </property>
       <property name="text">
        <string>回复后快速隐藏</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QKeySequenceEdit" name="replyKeyEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="showMySendCheck">
       <property name="toolTip">
        <string>回复后，显示自己发送的消息
如果在cqhttp的配置中开启了上报自身消息（report-self-message），则会显示两条，此时不建议开启</string>
       </property>
       <property name="text">
        <string>显示发送的消息</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="groupEmojiToImageCheck">
       <property name="toolTip">
        <string>因为风险控制，在群组发送表情包经常被屏蔽
开启此项后发送的表情包将以图片的形式发送</string>
       </property>
       <property name="text">
        <string>群表情包转图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
